src/report_formats/CSV_Hosts/CSV_Hosts.xsl

```diff
@@ -127,6 +127,30 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
       <xsl:value-of select="$cnt_fp"/><xsl:text>,</xsl:text>
 
       <xsl:value-of select="$cnt_high + $cnt_medium + $cnt_low + $cnt_log + $cnt_fp"/>
+      <xsl:text>,</xsl:text>
+
+      <xsl:value-of select="port_count/page"/>
+      <xsl:text>,</xsl:text>
+
+      <xsl:value-of select="count(detail[name/text() = 'App'])"/>
+      <xsl:text>,</xsl:text>
+
+      <xsl:variable name="traceroute" select="detail[name/text() = 'traceroute']"/>
+      <xsl:value-of select="string-length($traceroute) - string-length(translate($traceroute, ',', ''))"/>
+
+      <xsl:text>,</xsl:text>
+      <xsl:if test="detail[name/text() = 'Auth-SSH-Success']">
+        <xsl:text>SSH</xsl:text>
+      </xsl:if>
+      <xsl:if test="detail[name/text() = 'Auth-SMB-Success']">
```

Comment:
Shouldn't there be some kind of separator like a space between the different methods in case there are multiple successful authentication methods?

Reply:
I thought "SSHSMB" would be OK, but I guess a space would make them easier to parse.  In GSA it's just the same shield icon, repeated for each one.

Reply:
Done in 3fdbf41.